frontend/app/views/spree/shared/_option_values.html.erb

```diff
@@ -1,5 +1,3 @@
-<% cache base_cache_key + [ot_downcase_name, params_ot_downcase_name, filtering_params_cache_key, @taxon&.id] do %>
```

Comment:
I'm not sure this is really needed and it clashes with caching on a higher level here:

https://github.com/spree/spree/blob/f373a2dd3c6086499b1c600284b57b1f62afa859/frontend/app/views/spree/products/_filters_desktop.html.erb#L1

https://github.com/spree/spree/blob/ba4fbe01fd64ae86f7b3d828ebea0be4892edcda/frontend/app/views/spree/products/_filters_mobile.html.erb#L4

Reply:
@mad-eel so this should work as a classic Russian-doll caching, 

`filters_cache_key('desktop')` - caches the entire filter list
`cache base_cache_key + [ot_downcase_name, params_ot_downcase_name, filtering_params_cache_key, @taxon&.id]` - should cache a single filter? can be re-usable for both mobile/desktop - definitely this should be moved to a helper method and not be in the view, removing it completely hmmm we would need to think about it

Reply:
@damianlegawiec I understand, I'll reimplement it then 👍